stripe/src/main/java/com/stripe/android/net/StripeApiHandler.java

```diff
@@ -240,6 +261,63 @@ private static String formatURL(String url, String query) {
         return conn;
     }
 
+    private static void logTokenRequest(
+            @NonNull List<String> loggingTokens,
+            @Nullable RequestOptions options,
+            @Nullable LoggingResponseListener listener) {
+        if (options == null) {
+            return;
+        }
+
+        String apiKey = options.getPublishableApiKey();
+        if (apiKey.trim().isEmpty()) {
+            // if there is no apiKey associated with the request, we don't need to react here.
+            return;
+        }
+
+        String originalDNSCacheTTL = null;
+        Boolean allowedToSetTTL = true;
+
+        try {
```

Comment:
trying to factor this out felt really ugly, though there's a significant amount of copypasta with this and the fetch token method. I just didn't need nearly as much error handling, since I'm not going to surface any warnings for logging failures.